zebra-chain/src/parameters/network/testnet.rs

```diff
@@ -560,6 +578,9 @@ pub struct Parameters {
     target_difficulty_limit: ExpandedDifficulty,
     /// A flag for disabling proof-of-work checks when Zebra is validating blocks
     disable_pow: bool,
+    /// Whether to allow transactions with transparent outputs that spend coinbase inputs,
+    /// similar to `fCoinbaseMustBeShielded` in zcashd.
```

Comment:
Or

```suggestion
    /// Whether to allow transactions with transparent outputs to have inputs referencing
    /// coinbase transactions, similar to `fCoinbaseMustBeShielded` in zcashd.
```

Reply:
Applied alternative in https://github.com/ZcashFoundation/zebra/pull/9085/commits/1b69b498e7e806a8b9a5d98a4acdf17134980c7f.